azure/image/Dockerfile

```diff
@@ -70,5 +70,20 @@ RUN --mount=type=secret,id=AZURE_SUBSCRIPTION_ID \
     export AZURE_CLIENT_ID=$(cat /run/secrets/AZURE_CLIENT_ID) && \
     export AZURE_CLIENT_SECRET=$(cat /run/secrets/AZURE_CLIENT_SECRET) && \
     export AZURE_TENANT_ID=$(cat /run/secrets/AZURE_TENANT_ID) && \
```

Comment:
do we those still?

Reply:
Sorry I didn't get the question.

Reply:
😅 me neither, probably something I didn't understand initiall. nevermind